code/game/objects/items/devices/radio/radio.dm

```diff
@@ -221,6 +222,9 @@
 		return
 	if(!M.IsVocal())
 		return
+	if(radio_sounds.len) //Sephora - Radios make small static sounds now.
```

Comment:
This is honestly a nitpick, but:
```suggestion
	if(LAZYLEN(radio_sounds)) //Sephora - Radios make small static sounds now.
```
Never use A.len, instead opt in for length(A) proc.